libs/libgflags/Makefile

```diff
@@ -0,0 +1,46 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gflags
+PKG_VERSION:=2.2.2
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/gflags/gflags/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=34af2f15cf7367513b352bdcd2493ab14ce43692d2dcd9dfc499492966c64dcf
+
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILE:=COPYING.txt
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_BUILD_PARALLEL:=1
+HOST_BUILD_PARALLEL:=1
+CMAKE_OPTIONS:= \
+	-DGFLAGS_BUILD_SHARED_LIBS=ON \
+	-DGFLAGS_BUILD_STATIC_LIBS=ON
+CMAKE_INSTALL:=1
```

Comment:
Remove

Reply:
Not sure what's wrong here. CMAKE_INSTALL is for installing to staging_dir

Reply:
Implied by https://github.com/openwrt/openwrt/blob/master/include/cmake.mk#L3 ?

Reply:
https://github.com/openwrt/openwrt/blob/master/include/cmake.mk#L84 specifies Build/InstallDev . PKG_INSTALL specifies only Build/Install : https://github.com/openwrt/openwrt/blob/master/include/package.mk#L291